src/arbitrary.rs

```diff
@@ -76,6 +76,11 @@ impl Gen {
     {
         self.rng.gen_range(range)
     }
+
+    /// Create a random number uniformly distributed on the unit interval [0, 1)
+    pub fn gen_uniform(&mut self) -> f32 {
```

Comment:
That should probably be `gen_uniform_f32` or somesuch, since there could as well be versions for `f64`, and all the integer types.

Reply:
I guess one could make it polymorphic using num-traits..